django_project/changes/views/version.py

```diff
@@ -170,11 +170,30 @@ def get_context_data(self, **kwargs):
                 sponsors[sponsor.sponsorship_level].append(sponsor.sponsor)
 
         context['sponsors'] = sponsors
+        context['user_can_edit'] = False
+        context['user_can_delete'] = False
         project_slug = self.kwargs.get('project_slug', None)
         context['project_slug'] = project_slug
         if project_slug:
             context['project'] = Project.objects.get(slug=project_slug)
 
+        # lets check for specific user permissions here.
+        if self.request.user.is_staff:
```

Comment:
If it's me, I would use variables:
is_staff = self.request.user.is_staff
is_superuser = self.request.user.is_superuser
and so on, then put them in one if. Since all of them has the same context.

Reply:
Or put all of them in one `if` (without temp variable), I think python will do it better optimization.
Or just use `elif` so that we don't evaluate all of them. *I like this one most.

Reply:
Thanks 
Ill optimise when in the next PR for managing them in single file.